tests/ci/submit_azureml_pytest.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+'''
+submit_AzureML_pytest.py
+
+This will create create an Azure ML environment and submit pytest script as
+an experiment.
+
+'''
+
+import json
+
+# import time
+from azureml.core.authentication import AzureCliAuthentication
+from azureml.core import Workspace
+from azureml.core import Experiment
+from azureml.core.runconfig import RunConfiguration
+from azureml.core.conda_dependencies import CondaDependencies
+from azureml.core.runconfig import DEFAULT_CPU_IMAGE
+
+from azureml.core.script_run_config import ScriptRunConfig
+
+from azureml.core.compute import ComputeTarget, AmlCompute
+from azureml.core.compute_target import ComputeTargetException
+
+HELP_MSG = """
+This program sets up an AzureML environment and submits a pytest script on aml.
+$ python dir/setup_AzureML_script_to_submits_pytest.py
+"""
+
+
+def submit_exp():
```

Comment:
no need for separate function

also need argument parsing here too, in order to pass them onto run_pytest

Reply:
I pushed a stale copy.

Reply:
checked in stale copy